ctms/bin/acoustic_newsletters_mapping.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+"""Manage Acoustic newsletters mapping: add and remove from the db."""
+
+import argparse
+import os
+import sys
+
+from ctms import config
+from ctms.database import get_db_engine
+from ctms.models import AcousticNewsletterMapping
+
+
+def main(dbsession, test_args=None) -> int:
+    parser = argparse.ArgumentParser(
```

Comment:
I'm noticing the usage text is a bit sparse:
```
> python ctms/bin/acoustic_newsletters_mapping.py --help 

usage: acoustic_newsletters_mapping.py [-h] {add,remove,list} ...

Manage Acoustic Newsletter mapping

positional arguments:
  {add,remove,list}
```
```
> python ctms/bin/acoustic_newsletters_mapping.py add --help                                                                                            
usage: acoustic_newsletters_mapping.py add [-h] mapping

positional arguments:
  mapping

options:
  -h, --help  show this help message and exit
```

Could we maybe describe what a "mapping" is and its expected format and maybe sample usage of the script? I noticed you included this stuff in `docs/deployment_guide.md`, but it might be more useful here "closer" to the script.

Reply:
```
$ python ctms/bin/acoustic_newsletters_mapping.py --help
usage:
        python acoustic_newsletters_mapping.py add "test-pilot:sub_new_test_pilot"

Manage Acoustic Newsletter mapping

positional arguments:
  {add,remove,list}

options:
  -h, --help         show this help message and exit
```

This is how the `add` help looks now:
```
$ python ctms/bin/acoustic_newsletters_mapping.py add --help
usage: python acoustic_newsletters_mapping.py add "test-pilot:sub_new_test_pilot" add
       [-h] mapping

positional arguments:
  mapping     Add newsletter mapping specified as 'source:destination'

options:
  -h, --help  show this help message and exit